R/defStormsList.R

```diff
@@ -793,14 +813,15 @@ retrieveStorms <- function(database, filterNames, filterSeasons, removeTD) {
   #Removing NOT_NAMED storms
   indices <- indices[which(database$names[indices] != "NOT_NAMED")]
 
-  #Removing TD if removeTD == T
-  if (removeTD) {
-    suppressWarnings({
-      suppressWarnings(i <- which(apply(array(database$msw[, indices],
-                                              dim = c(dim(database$msw)[1], length(indices))),
-                                        2, max, na.rm = TRUE) >= 18 * knt2msC))
-      indices <- indices[i]
-    })
+  # Filter Storms if removeUnder is not NULL
+  if (!is.null(removeUnder)) {
+    # TODO change here
+    # suppressWarnings({
+    #   suppressWarnings(i <- which(apply(array(database$msw[, indices],
+    #                                           dim = c(dim(database$msw)[1], length(indices))),
+    #                                     2, max, na.rm = TRUE) >= database * knt2msC))
+    #   indices <- indices[i]
+    # })
   }
```

Comment:
To be done

Reply:
Ok but how would you like to implement it ? I mean what would you like the rules for this input to be ?

Reply:
Done, try this code

```
sds <- defStormsDataset(verbose = 0)


sts <- defStormsList(sds = sds,
                        loi = "Vanuatu",
                        removeUnder = 5,
                        verbose = 0)

sts <- defStormsList(sds = sds,
                        loi = "Vanuatu",
                        removeUnder = 6,
                        verbose = 0)
```